tests/tabs/test_list_all_tabs.py

```diff
@@ -0,0 +1,57 @@
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.keys import Keys
+
+from modules.browser_object import TabBar
+
+
+def test_list_all_tabs(driver: Firefox, screenshot):
+    # C134655
```

Comment:
i think we decided to do """""" type docs on test. also should indicate briefly what it does like other files

Reply:
On it